src/FakeItEasy.Analyzer.Shared/ArgumentConstraintTypeMismatchAnalyzer.cs

```diff
@@ -81,7 +81,7 @@ protected override void AnalyzeArgumentConstraintCore(string propertyFullName, S
                 if (nonNullableParameterType is object &&
                     constraintType.IsValueType &&
                     !constraintType.IsNullable() &&
-                    SymbolEqualityComparer.Default.Equals(constraintType, nonNullableParameterType))

```

Comment:
SymbolEqualityComparer isn't available in 2.10.0.

Reply:
Yes, that's what I suspected. Well, we managed without it before.